css/main.css

```diff
@@ -1,3 +1,139 @@
 /* Add your custom CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+h1, h2, h3 {
+font-family: 'Cinzel', serif;
+}
+
+body {
+    font-family: 'Quicksand', sans-serif;
+}
+
+ul {
+    list-style: none;
+}
+
+/* .li {
+display: flex;
+} */
+
+
+.hero-image {
+    background-image: url(https://images.unsplash.com/photo-1514533450685-4493e01d1fdc?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1934&q=80);
+
```

Comment:
To follow the design here, you will need to overlay this image (in smaller size) with the large golden shape in the background, and the beige small shape on top. Fo this, you'll need to use `position: absolute` on some elements (not all of them, but probably the small beige shape at least) and also `z-index` to define which one is on top / underneath.

Reply:
Hello Lucy, 

Thank you so much! I have made all the changes that you have requested. It was a great project.